app/mm-redux/constants/preferences.ts

```diff
@@ -55,115 +55,147 @@ const Preferences: Dictionary<any> = {
     TEAMS_ORDER: 'teams_order',
     CATEGORY_THEME: 'theme',
     THEMES: {
-        default: {
-            type: 'Mattermost',
-            sidebarBg: '#145dbf',
+        denim: {
+            type: 'Denim',
+            sidebarBg: '#1e325c',
             sidebarText: '#ffffff',
             sidebarUnreadText: '#ffffff',
-            sidebarTextHoverBg: '#4578bf',
-            sidebarTextActiveBorder: '#579eff',
+            sidebarTextHoverBg: '#28427b',
+            sidebarTextActiveBorder: '#5d89ea',
             sidebarTextActiveColor: '#ffffff',
-            sidebarHeaderBg: '#1153ab',
+            sidebarHeaderBg: '#192a4d',
             sidebarHeaderTextColor: '#ffffff',
-            onlineIndicator: '#06d6a0',
-            awayIndicator: '#ffbc42',
-            dndIndicator: '#f74343',
+            sidebarTeamBarBg: '#14213e',
+            onlineIndicator: '#3db887',
+            awayIndicator: '#ffbc1f',
+            dndIndicator: '#d24b4e',
             mentionBg: '#ffffff',
             mentionBj: '#ffffff',
```

Comment:
Shouldn't we remove this variable as it will no longer be used? I saw the selector no longer maps this one if `mentionBg` is not set.

Reply:
Yes, you're probably right.